packages/app/src/client/integrations/k8sMetrics/Show/InstallInstructions.tsx

```diff
@@ -71,6 +72,33 @@ export const InstallInstructions = ({
 }: InstallInstructionsProps) => {
   const dispatch = useDispatch();
 
+  const {
+    registerNotification,
+    unregisterNotification
+  } = useNotificationService();
+
+  const notifyError = useCallback(
+    (title: string, message: string) => {
+      const messageId = Math.floor(
+        Math.random() * Math.floor(100000)
+      ).toString();
+      const newNotification = {
+        id: messageId,
+        state: "error" as const,
+        title,
+        information: message,
+        handleClose: () =>
+          unregisterNotification({
+            id: messageId,
+            title: "",
+            information: ""
+          })
+      };
```

Comment:
I would like to at some point spend some time simplifying the notification service interface. There is a bunch of stuff in the interface that is not really used, but yet required.

Reply:
👍